pom.xml

```diff
@@ -486,7 +486,7 @@
         <version.org.jboss.metadata>16.1.0.Final</version.org.jboss.metadata>
         <version.org.jboss.mod_cluster>2.1.0.Final</version.org.jboss.mod_cluster>
         <version.org.jboss.narayana>7.2.2.Final</version.org.jboss.narayana>
-        <version.org.jboss.narayana.lra>1.0.2.Final-SNAPSHOT</version.org.jboss.narayana.lra>
+        <version.org.jboss.narayana.lra>1.0.1.Final</version.org.jboss.narayana.lra>
```

Comment:
We'll need to remember to upgrade this when we release "org.jboss.narayana.lra version 1.0.2.Final".